hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -765,40 +767,45 @@
 %>
 </table>
 <h2>Table Schema</h2>
+
 <table class="table table-striped">
+<%
+  Collection<HColumnDescriptor> families = new HTableDescriptor(table.getDescriptor()).getFamilies();
```

Comment:
Please use `ModifyableTableDescriptor` as `HTableDescriptor` is deprecated.

Reply:
Fixed, remove the unnecessary new HTableDescriptor and use ColumnFamilyDescriptor instead of HColumnDescriptor.
Thanks.